entity-service/src/main/kotlin/com/egm/stellio/entity/repository/SearchRepository.kt

```diff
@@ -41,7 +41,7 @@ interface SearchRepository {
         offset: Int,
         limit: Int,
         contexts: List<String>
-    ): Pair<Int, List<Entity>>
+    ): Triple<Int, List<Entity>, List<String>>
```

Comment:
it would be more readable and less error prone to create a simple data class containing just an `Entity` and a right